### ISSUE TITLE
Accessibility related issue in the forms

### ISSUE THREAD

**Issue by hannuheljala** [woocommerce/woocommerce#27664]
Hi!

I have noticed that woocommmerce.css declares this:

.woocommerce form .form-row input.input-text, .woocommerce form .form-row textarea {
box-sizing: border-box;
width: 100%;
margin: 0;
outline: 0;
line-height: normal;
}

It prevents the default outline being shown on focus. Using a border or outline style that obscures the focus ring causes problems for keyboard-only users. Can this “outline: 0;” be removed from all the Woocommerce related .css files? Thanks!

WCAG: https://www.w3.org/TR/WCAG20-TECHS/F78.html

**Comment by juliaamosova** [comment]
Hi @hannuheljala,

Thanks for reporting the issue. However, it is not fully clear what it is. Can you please provide step by step to replicate it locally? The screencast will help as well.

**Comment by hannuheljala** [comment]
Hi!

Steps to reproduce: Go to a checkout / login form / registration form of Woocommerce. In Chrome: Right click any of the input fields and select Inspect. In the Elements window right click the same input field, select Force State -> :focus. You can see no input focus is visible in the input field due this  outline: 0; in the woocommerce.css:

.woocommerce form .form-row input.input-text, .woocommerce form .form-row textarea {
    box-sizing: border-box;
    width: 100%;
    margin: 0;
    outline: 0;
    line-height: normal;
}

Another "normal" way to reproduce this is with TAB key go through the form, you can see that there is no focus indication ring visible.

Due to this Woocommerce does not met the WCAG criteria: "Failure of Success Criterion 2.4.7 due to styling element outlines and borders in a way that removes or renders non-visible the visual focus indicator"

https://www.w3.org/TR/WCAG20-TECHS/F78.html

**Comment by zhongruige** [comment]
Hi @hannuheljala,

Thank you for providing those details. However, I can’t reproduce it using the steps you provided. Everything is working as expected on my end. 

I tested this on both the latest FireFox (85.0) and Chrome (88.0.4324.96) with the following software versions:
* WordPress 5.6
* WooCommerce 4.9.2

**FireFox**

https://user-images.githubusercontent.com/71906536/106314316-cba80e80-6226-11eb-825b-89be3c1be2fc.mov

**Chrome**

https://user-images.githubusercontent.com/71906536/106314334-d367b300-6226-11eb-8d7a-876cdc741934.mov

Please provide us with more details about the issue which may help us to evaluate it further. Thank you!

**Comment by zhongruige** [comment]
Hi @hannuheljala ,

As a part of this repository’s maintenance, I am closing this issue due to inactivity. Please feel free to comment on it in case we missed something. We’d be happy to take another look.
